### DISCUSSION TITLE
Getting started with file browsing

### DISCUSSION THREAD

**Discussion by mcp292** [LunarVim/LunarVim#4507] [Q&A]
Hello! I'm new to LunarVim. Coming from Spacemacs with a lot of experience in Vim. No experience with Neovim.

When I run `lvim ~/`, for example, I can see my local files. I can navigate with `j` and `k` and open with `ENT`. But I have no idea how to go back up. I get stuck in whatever directory I entered.

I don't see any documentation about default file browsing in the LunarVim docs (just about adding plugins). I don't know how to find out what plugin I'm using at any given moment either, so I have no idea what to search with `:h`.

Can you point me in the right direction? What is the default file browser and where can I find documentation on the keybindings?

**Comment by brneor** [comment]
Optionally you can just `lvim` into current directory and open `NvimTree` with `<leader> e`.
